marathon/models/events.py

```diff
@@ -28,11 +26,7 @@ def __init__(self, event_type, timestamp, **kwargs):
         self.event_type = event_type  # All events have these two attributes
         self.timestamp = timestamp
         for attribute in self.KNOWN_ATTRIBUTES:
-            try:
-                self._set(attribute, kwargs[attribute])
-            except KeyError:
-                marathon.log.warn(
-                    'Unknown event attribute processing event {}: {}'.format(event_type, attribute))
+            self._set(attribute, kwargs.get(attribute))
```

Comment:
Agreed that for event simple json format is enough most of the time. However that will break the behavior of legacy code. If we still use mapping json to object way, it's better to remove the `KNOWN_ATTRIBUTES` limit, just check if the return is valid and map every field in json to attribute in object. This  might keep behavior same as before and keep code simple.